packages/x-components/src/components/suggestions/__tests__/base-suggestions.spec.ts

```diff
@@ -53,6 +59,64 @@ describe('testing Base Suggestions component', () => {
     expect(renderedSuggestions()).toHaveLength(suggestions.length);
   });
 
+  it('renders all suggestions filters if showFacets is true', () => {
```

Comment:
I did a big refactor, if you modify an old test, try to use the new way that we are applying in the new components, and use a render function. I wrote the code for you, just add documentation to the API and options section and check that everything is working fine and makes sense.